app/controllers/search.js

```diff
@@ -7,5 +7,11 @@ export default Controller.extend({
   },
 
   page: 1,
-  query: ''
+  query: '',
+
+  actions: {
+    setQuery(query) {
+      this.setProperties({ page: 1, query });
```

Comment:
Unrelated bugfix. Should go in it's own PR.

Reply:
Moved.